cms/server/contest/templates/task_submissions.html

```diff
@@ -168,33 +168,33 @@ <h2 style="margin: 40px 0 10px">{{ _("Previous submissions") }}</h2>
     {% set need_to_wait = tokens_info[2] is not none %}
     {% if can_play_token %}
         {% if tokens_info[0] == -1 %}
-            {{ _("Right now, you have infinite tokens available on this task.") }}
+            {% trans %}Right now, you have infinite tokens available on this task.{% endtrans %}
         {% elif tokens_info[0] == 1 %}
-            {{ _("Right now, you have one token available on this task.") }}
+            {% trans %}Right now, you have one token available on this task.{% endtrans %}
         {% else %}
-            {{ _("Right now, you have %(tokens)s tokens available on this task.") % { "tokens": tokens_info[0]} }}
+            {% trans tokens=tokens_info[0] %}Right now, you have {{ tokens }} tokens available on this task.{% endtrans %}
         {% endif %}
         {% if need_to_wait %}
             {% set t = tokens_info[2]|format_datetime_smart %}
-            {% raw _("But you have to wait until %(expiration_time)s to use them.") % {"expiration_time": t} %}
+            {% trans expiration_time=t %}But you have to wait until {{ expiration_time }} to use them.{% endtrans %}
         {% endif %}
         {% if tokens_info[1] is not none %}
             {% set t = tokens_info[1]|format_datetime_smart %}
-            {{ _("You will receive a new token at %(gen_time)s.") % {"gen_time": t} }}
+            {% trans gen_time=t %}You will receive a new token at {{ gen_time }}.{% endtrans %}
```

Comment:
Can you avoid the double assignment? Also above

Reply:
And below

Reply:
![](https://media.giphy.com/media/DgLsbUL7SG3kI/giphy.gif)

Reply:
Done.

Reply:
Would be nice to wrap lines now that we can :P

Reply:
You mean like this?